engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -112,6 +135,11 @@ public void setFontSize(int fontSize)
         }
     }
 
+    public String getUsedFont()
+    {
+        return transformer.getFontName();
+    }
+

```

Comment:
Is this used anywhere?

Reply:
It is only used for test purposes like `setStandardFont` and `setFontSize` methods.